itl/command-plugins-manubulon.conf

```diff
@@ -244,12 +244,23 @@ object CheckCommand "snmp-process" {
 		"-A" = {
 			set_if = "$snmp_process_use_params$"
 		}
+		"-m" = {
+			set_if = "$snmp_process_mem_usage$"
+			value = "$snmp_process_mem_thres$"
+		}
+		"-u" = {
+			set_if = "$snmp_process_cpu_usage$"
+			value = "$snmp_process_cpu_thres$"
+		}
 	}
 
 	vars.snmp_process_name = ".*"
 	vars.snmp_warn = 0
 	vars.snmp_crit = 0
 	vars.snmp_perf = true
 	vars.snmp_process_use_params = false
+	vars.snmp_process_mem_usage = false
+	vars.snmp_process_mem_thres = "0,0"
+	vars.snmp_process_cpu_usage = false
+	vars.snmp_process_cpu_thres = "0,0"
```

Comment:
Can you please rename `thres` to `threshold` and add the `description` field with additional details? Furthermore please update the documentation in doc/10-icinga-template-library.md.